tooling/profiler/src/cli/execution_flamegraph_cmd.rs

```diff
@@ -104,3 +132,22 @@ fn run_with_generator(
 
     Ok(())
 }
+
+fn is_brillig_entry_point(artifact: &ProgramArtifact) -> Result<(), CliError> {
```

Comment:
From the name, I expected it to return a `bool`:
```suggestion
fn ensure_brillig_entry_point(artifact: &ProgramArtifact) -> Result<(), CliError> {
```